src/Datadog.Trace.ClrProfiler.Managed/Integrations/HttpMessageHandlerIntegration.cs

```diff
@@ -70,6 +77,8 @@ public static class HttpMessageHandlerIntegration
                     {
                         // add distributed tracing headers to the HTTP request
                         SpanContextPropagator.Instance.Inject(scope.Span.Context, request.Headers.Wrap());
+
+                        scope.Span.Context.TraceContext.SamplingPriority = SamplingPriority.UserKeep;
```

Comment:
Why are we doing this? Our code should never directly set the `UserKeep` or `UserReject` values

Reply:
Good catch! This was test code, should not have made it into the PR. Removed in 5559addf84e345178e462a41072f6a47eda7d007.